src/relay_garbo.ts

```diff
@@ -0,0 +1,54 @@
+import { fileToArray, formField, formFields, svnInfo, writeln } from "kolmafia";
+import { get, set } from "libram";
+
+export function main(): void {
+  // handle updating values
+  const fields = formFields();
+  Object.keys(fields).forEach((field) => {
```

Comment:
```suggestion
  for (const field of Object.keys(fields)) {
```
From a high-level perspective, I'd like to include any code that's generating user-visible stuff to be inside React world and not generated server-side. But this is fine for now.